docker-compose.yml

```diff
@@ -8,12 +12,36 @@ services:
       - default
   qiskit-service:
     image: planqk/qiskit-service:latest
+    command: sh /app/startup.sh
     ports:
       - "5000:5000"
     environment:
       - REDIS_URL=redis://redis
+      - DATABASE_URL=sqlite:////data/app.db
+    volumes:
+      - exec_data:/data
     networks:
       - default
+
+  rq-worker:
+    image: planqk/qiskit-service:latest
+    command: rq worker --url redis://redis qiskit-service_execute
+    environment:
+      - REDIS_URL=redis://redis
+      - DATABASE_URL=sqlite:////data/app.db
+    volumes:
+      - exec_data:/data
+    deploy:
+      replicas: 4
+
+  
+  rq-dashboard:
+    image: eoranged/rq-dashboard
+    ports:
+      - "9181:9181"
+    environment:
+      - RQ_DASHBOARD_REDIS_URL=redis://redis:6379
+
 networks:
   default:
-    driver: bridge
\ No newline at end of file
+    driver: overlay
```

Comment:
with this, docker-compose up does not work anymore? what is the command to start the docker?

Reply:
On my machine it still works with the old command. But changing it back to "bridge" is also fine if it does the job.